publisher-sdk/src/androidTest/java/com/criteo/publisher/csm/CsmFunctionalTest.java

```diff
@@ -404,4 +430,11 @@ private void waitForIdleState() {
   private void loadBid(@NonNull AdUnit adUnit) {
     Criteo.getInstance().loadBid(adUnit, new ContextData(), ignored -> { /* no op */ });
   }
+
+  private void triggerMetricRequest() {
+    threadPoolExecutor.execute(() -> metricSendingQueueProducer.pushAllInQueue(metricRepository));
```

Comment:
Ah I didn't know about `MetricHelper`, will do.

Reply:
Though not gonna lie, it is an `enum` because initially there were three states. I can change it to `boolean` in the next PR where the consent is stored.